src/apps/administration/index.md

```diff
@@ -0,0 +1,41 @@
+# Administration Overview
+
+The Administration is the merchant interface of every Shopware store.
+
+It comprises UI for business logic, purchasing logic, data, and capabilities. It outlines all internal processes like product inventory tracking, order processing, and customer record management and gets input from frontend applications.
+
+## Build
+
+The [Administration module](https://developer.shopware.com/docs/concepts/framework/architecture/administration-concept) contains infrastructure and assets defined and used in the Administration user interface.
+
+You can modify the appearance and extend capabilities of the Administration by customizing its components such as [template files](https://developer.shopware.com/docs/guides/plugins/plugins/administration/writing-templates), [mixins](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-mixins), [fields](https://developer.shopware.com/docs/guides/plugins/plugins/administration/using-custom-fields), [filters](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-filter), [modules](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-module), [styling](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-styles), etc.
+
+This can be done via [apps](https://developer.shopware.com/docs/guides/plugins/apps/administration) or [plugins](https://developer.shopware.com/docs/guides/plugins/plugins/administration).
+
+### Rules
+
+You can create [custom rules](https://developer.shopware.com/docs/guides/plugins/plugins/framework/rule/add-custom-rules) to handle almost anything ecommerce-related for the customer, product or cart. It gives you control over what you want to happen if a specific condition is fulfilled.
+
+### Flow Builder
+
+[Flow Builder](https://developer.shopware.com/docs/guides/plugins/plugins/framework/flow/add-flow-builder-action#add-custom-action-in-administration) lets you build applications to automate a business process.
+
+### CMS
+
+Once the Administration component is built, the representation is handled by [CMS](https://developer.shopware.com/docs/guides/plugins/plugins/content/cms)
```

Comment:
This sounds like the cms handles the representation of the Administration. I'd describe it more as a management UI for the storefront content.